tests/conftest.py

```diff
@@ -1,103 +1,125 @@
 """Define global fixtures."""
 
-from os.path import join
 from pathlib import Path
 from pickle import load as _load
+from typing import List, Tuple
 
-from cobra import Metabolite, Model, Reaction
+from cobra import Metabolite, Model, Reaction, Solution
 from cobra.io import read_sbml_model
 from cobra.util import solver as sutil
 
 
 try:
     import pytest
-    import pytest_benchmark
 except ImportError:
     pytest = None
```

Comment:
I think you can just convert this whole thing to a simple pytest. We run our test suite with pytest, we have to assume it's installed.

Reply:
I agree.